waik/waik_provision_unattend.xml.erb

```diff
@@ -0,0 +1,158 @@
+            </ImageInstall>
+            <UserData>
+                <AcceptEula>true</AcceptEula>
+            </UserData>
+        </component>
+        <component name="Microsoft-Windows-International-Core-WinPE" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+            <SetupUILanguage>
+                <UILanguage>en-US</UILanguage>
+            </SetupUILanguage>
+            <% if @host.params['systemLocale'] -%>
+              <InputLocale><%= @host.params['systemLocale'] %></InputLocale>
+              <SystemLocale><%= @host.params['systemLocale'] %></SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage><%= @host.params['systemUILanguage'] %></UILanguage>
+              <UserLocale><%= @host.params['systemLocale'] %></UserLocale>
+            <% else -%>
+               <InputLocale>en-US</InputLocale>
+              <SystemLocale>en-US</SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage>en-US</UILanguage>
+              <UserLocale>en-US</UserLocale>
+            <% end -%>
+        </component>
+    </settings>
+    <settings pass="oobeSystem">
+        <component name="Microsoft-Windows-International-Core" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+            <% if @host.params['systemLocale'] -%>
+              <InputLocale><%= @host.params['systemLocale'] -%></InputLocale>
+              <SystemLocale><%= @host.params['systemLocale'] -%></SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage><%= @host.params['systemUILanguage'] -%></UILanguage>
+              <UserLocale><%= @host.params['systemLocale'] -%></UserLocale>
+            <% else -%>
+               <InputLocale>en-US</InputLocale>
+              <SystemLocale>en-US</SystemLocale>
+              <UILanguageFallback>en-US</UILanguageFallback>
+              <UILanguage>en-US</UILanguage>
+              <UserLocale>en-US</UserLocale>
+            <% end -%>
+        </component>
+        <component name="Microsoft-Windows-Shell-Setup" processorArchitecture="amd64" publicKeyToken="31bf3856ad364e35" language="neutral" versionScope="nonSxS" xmlns:wcm="http://schemas.microsoft.com/WMIConfig/2002/State" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+            <UserAccounts>
+                <AdministratorPassword>
+                    <PlainText>false</PlainText>
+                    <Value><%= Base64.encode64(Encoding::Converter.new("UTF-8", "UTF-16LE",:undef => nil).convert(Base64.decode64(root_pass)+"AdministratorPassword")).delete!("\n").chomp %></Value>
```

Comment:
I'd prefer not to ship a template that only works when safemode's off, as I strongly recommend people always leave it enabled - it's such a security risk if you have non-admin users otherwise.  So my preference is to do something/anything, in Foreman to make it possible to run in safemode.